### ISSUE TITLE
Feature request: exportation of notes

### ISSUE THREAD

**Issue by flaviozluca** [Intermesh/groupoffice#602]
Dear friends,

I didn't find a way to export notes all at once. I use them extensively... Is there some way to back them up with a single command or query...? Only way I found to have a backup is to print each and every one, which is a lot of work.

If not, can you add that as a feature request, please? 

Would be nice, even, for other groupware objects, such as contacts, events, to have some type of per user backup...

Thanks.

**Comment by derjoachim** [comment]
Since you want to be able to back up your notebooks, I created a script that exports the selected notebook as CSV. Since an import would be nice too, I added that functionality as well.

Both the CSV export and CSV import can be called from the menu on top of the note grid:

![image](https://user-images.githubusercontent.com/1264970/92250576-ecfd0c80-eecb-11ea-8e45-4e410904b3d9.png)

It will be rolled out in the next release.
